Tests/TestUtilities/DummyExeHelper.cs

```diff
@@ -0,0 +1,146 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DummyExeHelper.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.CSharp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.CodeDom.Compiler;
+using System.IO;
+
+namespace TestUtilities
+{
+    /// <summary>
+    /// Creates dummy executables that log the input parameters and return a specified
```

Comment:
Does what it says on the tin. To test the bootstrapper, we need to have dummy pre- and post- processor exes with the correct file names that the bootstrapper can invoke. This helper classes uses the .Net framework to compile simple dummy exes on the fly.